CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance and religion.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
```

Comment:
I think the right writing is includes, in plural

Reply:
`include` is a verb, `s` is necessary after `he`, `she` and `it`